### PR TITLE
Remove default values in `new_bb1_control`

### PR DESCRIPTION
**Pull request by tachikoma-li**

### CODE REVIEW

File: qctrlopencontrols/driven_controls/predefined.py [hunk 1]
```diff
@@ -183,8 +180,8 @@ def new_bb1_control(
         Defaults to :math:`2\pi`.
```
Comment (charmasaur) [inline qctrlopencontrols/driven_controls/predefined.py:180]: Remove

File: qctrlopencontrols/driven_controls/predefined.py [hunk 2]
```diff
@@ -164,9 +161,9 @@ def new_primitive_control(
 
 def new_bb1_control(
     rabi_rotation: float,
-    azimuthal_angle: float = 0.0,
-    maximum_rabi_rate: float = 2.0 * np.pi,
-    **kwargs
+    azimuthal_angle: float,
```
Comment (charmasaur) [inline qctrlopencontrols/driven_controls/predefined.py:164]: The docs still state that this is optional. Either update the docs, or keep this defaulting to zero (I think that would be quite reasonable, FWIW).

File: qctrlopencontrols/driven_controls/predefined.py (hunk 1 above)
Comment (tachikoma-li) [reply qctrlopencontrols/driven_controls/predefined.py]: removed

File: qctrlopencontrols/driven_controls/predefined.py (hunk 2 above)
Comment (tachikoma-li) [reply qctrlopencontrols/driven_controls/predefined.py]: Agreed. I also add 0 back as the default value of  `azimuthal_angle`  in `new_primitive_control`

**Review by charmasaur — APPROVED** [review]

**Pull request merged**